lib/drawer/index.js

```diff
@@ -1,33 +1,29 @@
-import { delimiters } from './delimiters';
 import { metaballs } from '../metaballs';
-import axesFactory from './axes';
+import { delimiters } from './delimiters';
 import dropsFactory from './drops';
 import labelsFactory from './labels';
 import lineSeparatorFactory from './lineSeparator';
+import {drawTopAxis,drawBottomAxis,boolOrReturnValue} from './xAxis';
 
 export default (svg, dimensions, scales, configuration) => {
     const defs = svg.append('defs');
-
+    
     defs.append('clipPath')
         .attr('id', 'drops-container-clipper')
         .append('rect')
             .attr('id', 'drops-container-rect')
             .attr('width', dimensions.width)
             .attr('height', dimensions.height + configuration.margin.top + configuration.margin.bottom);
-
+    
     const labelsContainer = svg
         .append('g')
         .classed('labels', true)
         .attr('transform', `translate(0, ${configuration.lineHeight})`);
 
     const chartContainer = svg.append('g')
-        .attr({
-            class: 'chart-wrapper',
-            transform: `translate(${configuration.labelsWidth + configuration.labelsRightMargin}, 55)`,
-        });
-
-    const axesContainer = chartContainer.append('g').classed('axes', true);
-
+        .attr('class', 'chart-wrapper')
```

Comment:
The object I did included all the drawer files together . I think It will need a separate issue for that